test/dlc_tests/conftest.py

```diff
@@ -129,16 +129,21 @@
     "feature_s3_plugin_present": {NightlyFeatureLabel.AWS_S3_PLUGIN_INSTALLED.value},
 }
 
+
 @pytest.fixture(autouse=True)
-def skip_s3plugin_test(request, pytorch_training):
+def skip_s3plugin_test(request):
```

Comment:
A fix has been merged into the master. We can revert these changes and update this branch with the Master.

Reply:
Done